### ISSUE TITLE
Add option to ignore blinking cursors / carets

### ISSUE THREAD

**Issue by nextlevelbeard** [webdriverio/visual-testing#691]
**Is your feature request related to a problem? Please describe.**
Cannot properly compare visual info on inputs due to a blinking cursor on most inputs

**Describe the solution you'd like**
A simple option that can be defaulted in the config or passed to a certain check command as a boolean value to "ignore" the blinking cursor. Potentially enhance type `CheckScreenMethodOptions` to allow boolean property `ignoreCarets` that styles every input with `caretColor: 'transparent'` style.

**Describe alternatives you've considered**
With WDIO, set element's s caretColor style to transparent before comparing:
```ts
await browser.execute(
    function (this: Document, e: HTMLElement | WebdriverIO.Element | ChainablePromiseElement) {
      (e as HTMLElement).style.caretColor = 'transparent'
    },
    await email
  )
```

**Additional context**
N/A

**Comment by wswebcreation** [comment]
Thanks @nextlevelbeard , something I'll add in one of the next releases

**Comment by wswebcreation** [comment]
This will be released with https://github.com/webdriverio/visual-testing/pull/688

**Comment by wswebcreation** [comment]
@nextlevelbeard 

It's fixed in version [`6.3.0`](https://github.com/webdriverio/visual-testing/releases/tag/%40wdio%2Fvisual-service%406.3.0)
